### ISSUE TITLE
regression in 2.12.x-release-website-archives

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-dev#89]
2 failures in a row:
- https://scala-ci.typesafe.com/job/scala-2.12.x-release-website-archives/135/consoleFull
- https://scala-ci.typesafe.com/job/scala-2.12.x-release-website-archives/136/consoleFull

**Comment by SethTisue** [comment]
the failure can be reproduced simply by hitting rebuild on the release-website-archives job, without having to wait for the entire nightly build process.

**Comment by SethTisue** [comment]
on chara:

```
chara% ls /home/linuxsoft/archives/scala/nightly/2.12.x/api/2.12.0-1564f28-nightly/jars
jline-2.12.1-javadoc.jar               scala-parser-combinators_2.12.0-1564f28-nightly-1.0.4-javadoc.jar
scala-compiler-2.12.0-1564f28-nightly-javadoc.jar  scala-reflect-2.12.0-1564f28-nightly-javadoc.jar
scala-library-2.12.0-1564f28-nightly-javadoc.jar   scala-swing_2.12.0-1564f28-nightly-2.0.0-M2-javadoc.jar
scalap-2.12.0-1564f28-nightly-javadoc.jar      scala-xml_2.12.0-1564f28-nightly-1.0.5-javadoc.jar
```

note the presence of module javadoc jars like `scala-parser-combinators_2.12.0-1564f28-nightly-1.0.4-javadoc.jar`

but then

```
chara% ls /home/linuxsoft/archives/scala/nightly/2.12.x/api/2.12.0-a4b7e1c-nightly/jars
jline-2.12.1-javadoc.jar               scalap-2.12.0-a4b7e1c-nightly-javadoc.jar
scala-compiler-2.12.0-a4b7e1c-nightly-javadoc.jar  scala-reflect-2.12.0-a4b7e1c-nightly-javadoc.jar
scala-library-2.12.0-a4b7e1c-nightly-javadoc.jar
```

**Comment by SethTisue** [comment]
at e.g. https://scala-ci.typesafe.com/job/scala-2.12.x-release-package-unix/175/consoleFull (a green run from a few weeks ago) we see:

```
[info] downloading https://scala-ci.typesafe.com/artifactory/scala-release-temp/org/scala-lang/modules/scala-parser-combinators_2.12.0-1564f28-nightly/1.0.4/scala-parser-combinators_2.12.0-1564f28-nightly-1.0.4-javadoc.jar ...
```

and then later during zipping:

```
scala-2.12.0-1564f28-nightly/api/jars/scala-parser-combinators_2.12.0-1564f28-nightly-1.0.4-javadoc.jar
```

whereas in a recent run we see:

```
[warn]  [FAILED     ] org.scala-lang.modules#scala-parser-combinators_2.12.0-a4b7e1c-nightly;1.0.4!scala-parser-combinators_2.12.0-a4b7e1c-nightly.jar(doc):  (0ms)
[warn] ==== private-repo: tried
[warn]   https://scala-ci.typesafe.com/artifactory/scala-release-temp/org/scala-lang/modules/scala-parser-combinators_2.12.0-a4b7e1c-nightly/1.0.4/scala-parser-combinators_2.12.0-a4b7e1c-nightly-1.0.4-javadoc.jar
[warn] ==== jcenter-cache: tried
[warn]   https://scala-ci.typesafe.com/artifactory/jcenter/org/scala-lang/modules/scala-parser-combinators_2.12.0-a4b7e1c-nightly/1.0.4/scala-parser-combinators_2.12.0-a4b7e1c-nightly-1.0.4-javadoc.jar
[warn] ==== typesafe-ivy-releases: tried
[warn]   https://dl.bintray.com/typesafe/ivy-releases/org.scala-lang.modules/scala-parser-combinators_2.12.0-a4b7e1c-nightly/1.0.4/docs/scala-parser-combinators_2.12.0-a4b7e1c-nightly-javadoc.jar
[warn] ==== sbt-plugin-releases: tried
[warn]   https://dl.bintray.com/sbt/sbt-plugin-releases/org.scala-lang.modules/scala-parser-combinators_2.12.0-a4b7e1c-nightly/1.0.4/docs/scala-parser-combinators_2.12.0-a4b7e1c-nightly-javadoc.jar
[warn] ==== public: tried
[warn]   https://repo1.maven.org/maven2/org/scala-lang/modules/scala-parser-combinators_2.12.0-a4b7e1c-nightly/1.0.4/scala-parser-combinators_2.12.0-a4b7e1c-nightly-1.0.4-javadoc.jar
[warn] ==== local: tried
[warn]   /home/jenkins/workspace/scala-2.12.x-release-package-unix/.ivy2/local/org.scala-lang.modules/scala-parser-combinators_2.12.0-a4b7e1c-nightly/1.0.4/docs/scala-parser-combinators_2.12.0-a4b7e1c-nightly-javadoc.jar
```

and indeed a javadoc jar is present at https://scala-ci.typesafe.com/artifactory/scala-release-temp/org/scala-lang/modules/scala-parser-combinators_2.12.0-1564f28-nightly/1.0.4/ but missing from https://scala-ci.typesafe.com/artifactory/scala-release-temp/org/scala-lang/modules/scala-parser-combinators_2.12.0-a4b7e1c-nightly/1.0.4/

**Comment by SethTisue** [comment]
farther upstream in the release process, in an older build like https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-bootstrap/308/artifact/logs/builds/*view*/ we see:

```
[info] Packaging /home/jenkins/workspace/scala-2.12.x-integrate-bootstrap/scala-parser-combinators/target/scala-2.12.0-1564f28-nightly/scala-parser-combinators_2.12.0-1564f28-nightly-1.0.4-sources.jar ...
[info] Packaging /home/jenkins/workspace/scala-2.12.x-integrate-bootstrap/scala-parser-combinators/target/scala-2.12.0-1564f28-nightly/scala-parser-combinators_2.12.0-1564f28-nightly-1.0.4-javadoc.jar ...
[info] Done packaging.
[info] Wrote /home/jenkins/workspace/scala-2.12.x-integrate-bootstrap/scala-parser-combinators/target/scala-2.12.0-1564f28-nightly/scala-parser-combinators_2.12.0-1564f28-nightly-1.0.4.pom
[info] :: delivering :: org.scala-lang.modules#scala-parser-combinators_2.12.0-1564f28-nightly;1.0.4 :: 1.0.4 :: release :: Thu Feb 18 01:49:16 UTC 2016
[info]  delivering ivy file to /home/jenkins/workspace/scala-2.12.x-integrate-bootstrap/scala-parser-combinators/target/scala-2.12.0-1564f28-nightly/ivy-1.0.4.xml
[info] Done packaging.
[warn] bnd: Unused Private-Package instructions, no such package(s) on the class path: [org.scala-lang.modules.scala-parser-combinators.*]
[info]  published scala-parser-combinators_2.12.0-1564f28-nightly to https://scala-ci.typesafe.com/artifactory/scala-release-temp/org/scala-lang/modules/scala-parser-combinators_2.12.0-1564f28-nightly/1.0.4/scala-parser-combinators_2.12.0-1564f28-nightly-1.0.4.pom
[info]  published scala-parser-combinators_2.12.0-1564f28-nightly to https://scala-ci.typesafe.com/artifactory/scala-release-temp/org/scala-lang/modules/scala-parser-combinators_2.12.0-1564f28-nightly/1.0.4/scala-parser-combinators_2.12.0-1564f28-nightly-1.0.4.jar
[info]  published scala-parser-combinators_2.12.0-1564f28-nightly to https://scala-ci.typesafe.com/artifactory/scala-release-temp/org/scala-lang/modules/scala-parser-combinators_2.12.0-1564f28-nightly/1.0.4/scala-parser-combinators_2.12.0-1564f28-nightly-1.0.4-sources.jar
[info]  published scala-parser-combinators_2.12.0-1564f28-nightly to https://scala-ci.typesafe.com/artifactory/scala-release-temp/org/scala-lang/modules/scala-parser-combinators_2.12.0-1564f28-nightly/1.0.4/scala-parser-combinators_2.12.0-1564f28-nightly-1.0.4-javadoc.jar
```

which includes javadoc. but in https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-bootstrap/317/artifact/logs/builds/*view*/ the `-sources` lines are there but the adjacent `-javadoc` lines are missing:

```
[info] Packaging /home/jenkins/workspace/scala-2.12.x-integrate-bootstrap/scala-parser-combinators/target/scala-2.12.0-a4b7e1c-nightly/scala-parser-combinators_2.12.0-a4b7e1c-nightly-1.0.4-sources.jar ...
[info] Done packaging.
[info] Wrote /home/jenkins/workspace/scala-2.12.x-integrate-bootstrap/scala-parser-combinators/target/scala-2.12.0-a4b7e1c-nightly/scala-parser-combinators_2.12.0-a4b7e1c-nightly-1.0.4.pom
[info] :: delivering :: org.scala-lang.modules#scala-parser-combinators_2.12.0-a4b7e1c-nightly;1.0.4 :: 1.0.4 :: release :: Wed Feb 24 01:48:54 UTC 2016
[info]  delivering ivy file to /home/jenkins/workspace/scala-2.12.x-integrate-bootstrap/scala-parser-combinators/target/scala-2.12.0-a4b7e1c-nightly/ivy-1.0.4.xml
[warn] bnd: Unused Private-Package instructions, no such package(s) on the class path: [org.scala-lang.modules.scala-parser-combinators.*]
[info]  published scala-parser-combinators_2.12.0-a4b7e1c-nightly to https://scala-ci.typesafe.com/artifactory/scala-release-temp/org/scala-lang/modules/scala-parser-combinators_2.12.0-a4b7e1c-nightly/1.0.4/scala-parser-combinators_2.12.0-a4b7e1c-nightly-1.0.4.pom
[info]  published scala-parser-combinators_2.12.0-a4b7e1c-nightly to https://scala-ci.typesafe.com/artifactory/scala-release-temp/org/scala-lang/modules/scala-parser-combinators_2.12.0-a4b7e1c-nightly/1.0.4/scala-parser-combinators_2.12.0-a4b7e1c-nightly-1.0.4.jar
[info]  published scala-parser-combinators_2.12.0-a4b7e1c-nightly to https://scala-ci.typesafe.com/artifactory/scala-release-temp/org/scala-lang/modules/scala-parser-combinators_2.12.0-a4b7e1c-nightly/1.0.4/scala-parser-combinators_2.12.0-a4b7e1c-nightly-1.0.4-sources.jar
```

**Comment by SethTisue** [comment]
culprit is probably https://github.com/scala/scala/commit/c4fc2fd42457a87fea9e7af94e0ba2f57e533854 — perhaps “skip docs on first module build” is a clue

**Comment by SethTisue** [comment]
merged scala/scala#5007, so let's see what happens tonight

**Comment by SethTisue** [comment]
several greens in a row now at https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-release-main/
